### PR TITLE
changing if statements to elif

### PR DESCRIPTION
**Pull request by abit2**
@DanRyanIrish

### CODE REVIEW

**Comment by DanRyanIrish** [comment]
Can you finish off the outer layer of ifs and elifs with an else: raise TypeError which prints our the item type?

**Comment by DanRyanIrish** [comment]
The same for the inner if...elif starting at https://github.com/abit2/cube/blob/3ea293d4f8153e802c50ec35248609d8a44ada78/sunpycube/cube/cube_utils.py#L526
would be good too

**Comment by abit2** [comment]
but that if condition is inside if condition 
```
if ---------:
    if -----------:
    
    elif ---------:

**Comment by DanRyanIrish** [comment]
Well the more important one of this case we're dealing with is the outer if...elif..nest

**Comment by DanRyanIrish** [comment]
Do that for now.

File: sunpycube/cube/cube_utils.py [hunk 1]
```diff
@@ -549,6 +549,9 @@ def get_cube_from_sequence(cubesequence, item):
                     data[i] = cube[item[2::]]
             result = cubesequence._new_instance(
                 data, meta=cubesequence.meta, common_axis=cubesequence.common_axis)
+        elif:
```
Comment (DanRyanIrish) [inline sunpycube/cube/cube_utils.py]: should be ```else```, not ```elif```

Comment (abit2) [reply sunpycube/cube/cube_utils.py]: oh :P

File: sunpycube/cube/cube_utils.py [hunk 2]
```diff
@@ -549,6 +549,9 @@ def get_cube_from_sequence(cubesequence, item):
...
+            result = None
```
Comment (DanRyanIrish) [inline sunpycube/cube/cube_utils.py]: Can delete this line setting result to None.  The next line raises an error anyway.

**Review by DanRyanIrish — APPROVED** [review]

**Pull request merged**